vm-setup/roles/v1aX_integration_test/tasks/node_reuse.yml

```diff
@@ -441,28 +441,7 @@
         spec:
           replicas: 3
 
-  - name: Wait until all "{{ NUMBER_OF_ALL_BMH }}" BMHs' are provisioned.
-    k8s_info:
-      api_version: metal3.io/v1alpha1
-      kind: BareMetalHost
-      namespace: "{{ NAMESPACE }}"
-      kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    register: bmhs
-    retries: 200
-    delay: 20
-    until:
-      - bmhs is succeeded
-      - bmhs.resources | filter_provisioning("provisioned") | length == (NUMBER_OF_ALL_BMH | int)
-
-  - name: Wait until all "{{ NUMBER_OF_ALL_BMH }}" machines become running.
-    k8s_info:
-      api_version: cluster.x-k8s.io/{{ CAPI_VERSION }}
-      kind: machines
-      namespace: "{{ NAMESPACE }}"
+  - name: Verify that all machines are provisioned and running.
+    include_tasks: verify_provisioned.yml
+    vars:
       kubeconfig: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
-    register: machines
-    retries: 200
-    delay: 20
-    until:
-      - machines is succeeded
-      - machines.resources | filter_phase("running") | length == (NUMBER_OF_ALL_BMH | int)
```

Comment:
**NUMBER_OF_ALL_BMH** defined at the beginning of this file can be safely removed I believe, since it is covered by a more consistent var called **NUMBER_OF_BMH**?

Reply:
Yes I think you are correct. Not sure why we have both. I'll update!